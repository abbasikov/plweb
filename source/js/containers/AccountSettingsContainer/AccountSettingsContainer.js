import React from 'react';
import { browserHistory } from 'react-router';
import routes from '../../routes';

export default class AccountContainer extends React.Component {

    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div className="page-container">
                <div className="page-content">
                    <div className="content-wrapper">
                        <div className="panel panel-flat ">
                            <div className="panel-body panel-post">
                                this is account settings container
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        );
    }
}