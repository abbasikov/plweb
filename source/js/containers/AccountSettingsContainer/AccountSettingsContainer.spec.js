import React from 'react';
import AccountSettingsContainer from './AccountSettingsContainer';
import renderer from 'react-test-renderer';

describe('AccountSettingsContainer', function() {
    it('#render', function() {
        const component = renderer.create(
            <AccountSettingsContainer />
        );
        var tree = component.toJSON();
        expect(tree).toMatchSnapshot();

    });
});