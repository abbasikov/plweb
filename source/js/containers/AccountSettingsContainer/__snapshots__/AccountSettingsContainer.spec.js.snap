// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountSettingsContainer #render 1`] = `
<div
  className="page-container"
>
  <div
    className="page-content"
  >
    <div
      className="content-wrapper"
    >
      <div
        className="panel panel-flat "
      >
        <div
          className="panel-body panel-post"
        >
          this is account settings container
        </div>
      </div>
    </div>
  </div>
</div>
`;
